docker/install_node.sh

```diff
@@ -14,10 +14,17 @@
 # limitations under the License.
 
 #!/bin/bash
+# Check if node is installed. If it is, skip installation.
+if [ -d "../oppia_tools/node-16.13.0" ]; then
+    echo "Node.js is already installed. Skipping the installation."
+    exit 0
+fi
+
+# Install node.js if it is not installed.
 OS_NAME=$(uname)
 echo "Installing Node.js..."
 
-if [ "$OS_NAME" = "Windows" ]; then
+if [ "$OS_NAME" = "Windows" ] || [ $(uname -o) = "Msys" ]; then
```

Comment:
Could you please add comment explaining why `$(uname -o) = "Msys"` was added?

Reply:
uname returns whole output MYSYS-22.10.12, which might change in future as version updates. uname -o returns operating system thus only Msys. We can use `uname -o` instead of just ` uname ` in OS_NAME, but I couldn't verify if it gives required output in Windows 10 and Darwin.

Reply:
Also checked, pop-os returns `GNU/Linux` in `uname -o`, I think changing OS_NAME will make it more complicated. Also adding comment in line 27 for same.